chromium_src/components/search_engines/search_engines_pref_names.h

```diff
@@ -11,6 +11,7 @@
 namespace prefs {
 
 extern const char kDefaultSearchProviderByExtension[];
+extern const char kOtherSearchEnginesEnabled[];
```

Comment:
The name sounds confusing - not really clear what are "other" search engines.
Mb something like `kAddOpenSearchEngines`?

Reply:
Yes that name is confusing now that you mention it, updated.